default.nix

```diff
@@ -0,0 +1,9 @@
+{pkgs ? import <nixpkgs> {}}: let
+  lib = pkgs.lib;
+  matchbox = pkgs.buildGoModule {
```

Comment:
Can you make buidlGoModule be the main block in the file, instead of in the let expression?

Reply:
https://github.com/mgrzybek/matchbox/commit/870f6a2b04387da67416824d3e9f98036ae6cfc6